### DISCUSSION TITLE
Support for Tailwind and JIT mode

### DISCUSSION THREAD

**Discussion by hsheikhali1** [nvh95/jest-preview#140] [Q&A] [answered]
Would there or is there any support for CSS libraries like tailwindcss?

**Comment by nvh95** [comment, marked as answer]
Hi @hsheikhali1. Absolutely yes.
We will support TailwindCSS in the next release. Hopefully it will be out by the end of this month.
In the mean time, you can do a work around by exporting the CSS and import it to jest setup file. You can see the full instruction here:

https://github.com/nvh95/jest-preview/pull/127#issuecomment-1153348190

Let me know if it works for you.

**Reply by hsheikhali1** [reply]
@nvh95 Thanks for responding! this is awesome btw!!

**Comment by nvh95** [comment]
Tailwind CSS support will land in `0.2.5` at the latest at the end of this month (2022 June).

For now `jest-preview@0.2.5-alpha.0` is out and it already supports **BASIC** TailwindCSS out of the box (I tried on a CRA project and it works fine). You can try it out as follow:

```
npm install jest-preview@alpha
```

Don't forget to import your global CSS file to jest setup file, for example:

```js
// setupTest.js
import "./index.css"; // contains @tailwind base; @tailwind components; @tailwind utilities;
```

Please let me know if you guys spot any bugs. Thanks.
